### PR TITLE
Update unified customization models according to new scheme

### PR DESCRIPTION
**Pull request by DavDo**
MOB 2015

**Jira issue:**
https://glia.atlassian.net/browse/MOB-2015

**Additional info:**
related changes - https://github.com/salemove/mobile-sdk-unified/pull/26

### CODE REVIEW

**Review by gugalo — APPROVED** [review]
I didn't find anything wrong

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/secureconversations/SecureConversationsWelcomeScreenTheme.kt [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
+    val disabledSendButtonTheme: ButtonTheme? = null,
+    val loadingSendButtonTheme: ButtonTheme? = null,
+    val activityIndicatorColorTheme: ColorTheme? = null,//
+    val messageWarningTheme: TextTheme? = null,//
+    val messageWarningIconColorTheme: ColorTheme? = null,//
+    val filePickerButtonTheme: ColorTheme? = null,//
+    val filePickerButtonDisabledTheme: ColorTheme? = null,//
+    val attachmentListTheme: FileUploadBarTheme? = null,
+    val pickMediaTheme: AttachmentsPopupTheme? = null,
+    val backgroundTheme: ColorTheme? = null//
```
Comment (andrews-moc) [inline widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/secureconversations/SecureConversationsWelcomeScreenTheme.kt:29]: I think we can remove the `//` at the end of each line. Or add descriptions. 😉

**Review by andrews-moc — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/secureconversations/SecureConversationsWelcomeScreenTheme.kt (hunk 1 above)
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/secureconversations/SecureConversationsWelcomeScreenTheme.kt:29]: Thank you, those were leftovers.

**Review by AndriiHorishniiMOC — APPROVED** [review]

**Comment by DavDo** [comment]
!squash

**Review by Pelkar — APPROVED** [review]
LGTM 👍

**Comment by DavDo** [comment]
!merge

**Pull request merged**